corehq/apps/export/models/new.py

```diff
@@ -935,6 +935,7 @@ def put_item(self, path):
         item = ExportItem(
             path=path,
             label='.'.join(map(lambda node: node.name, path)),
+            last_occurrences={INFERRED_OCCURRENCE: INFERRED_OCCURRENCE}
```

Comment:
this is somewhat ugly. open to ideas. i needed a way to express that this item didn't belong to an app / build version pair and that it was infered